src/main/java/net/snowflake/ingest/streaming/internal/AbstractRowBuffer.java

```diff
@@ -200,7 +205,13 @@ public InsertValidationResponse insertRows(
       for (Map<String, Object> row : rows) {
         Set<String> inputColumnNames = verifyInputColumns(row, null, tempRowCount);
         tempRowsSizeInBytes +=
-            addTempRow(row, tempRowCount, rowBuffer.tempStatsMap, inputColumnNames, tempRowCount);
+            addTempRow(
+                row,
+                tempRowCount,
+                rowBuffer.tempStatsMap,
+                inputColumnNames,
+                tempRowCount,
+                null /* error */);
```

Comment:
the called method now has to know that a null can be passed in, adds to unnecessary code complexity. We should either mark the type as Optional to make it clear a null can be passed in, or just pass in an error even if its not going to be used. The latter is preferable IMO.

Reply:
Make sense, passed a dummy error.